main.tf

```diff
@@ -442,19 +442,23 @@ module "ecs_cluster_asg" {
 resource "aws_s3_bucket" "binaries" {
   bucket        = "${data.aws_caller_identity.current.account_id}-${data.aws_region.current.name}-binaries"
   force_destroy = true
+  tags = local.tags
+}
 
-  server_side_encryption_configuration {
```

Comment:
Fixes:
> │ Warning: Argument is deprecated
│ 
│   with aws_s3_bucket.binaries,
│   on main.tf line 442, in resource "aws_s3_bucket" "binaries":
│  442: resource "aws_s3_bucket" "binaries" {